Mathlib/RingTheory/SimpleRing/TensorProduct.lean

```diff
@@ -0,0 +1,362 @@
+/-
+Copyright (c) 2025 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Yunzhou Xie
+-/
+import Mathlib.Algebra.Central.Basic
+import Mathlib.LinearAlgebra.Basis.VectorSpace
+import Mathlib.RingTheory.Flat.Basic
+import Mathlib.RingTheory.TwoSidedIdeal.SpanAsSum
+
+/-!
+# Tensor product of simple algebras
+
+In this file, we show that the tensor product between a simple algebra and a central simple algebra
+is simple. This a pre-requist for proving the group law of the Brauer group, in particular it helps
+show that tensor product of two central simple algebras is central simple.
+
+## Main Results
+
+* `TensorProduct.nontrivial`: The tensor product of two non-trivial algebras is non-trivial.
+* `TensorProduct.simple`: The tensor product of a simple algebra and a central simple algebra
+  is simple.
+
+## References
+
+* [StackProject 074B](https://stacks.math.columbia.edu/tag/074B)
+
+## Tags
+Noncommutative algebra, tensor product, simple algebra, central simple algebra
+
+-/
+
+variable (K : Type*) [Field K]
+
+universe u v
+
+open scoped TensorProduct
+
+variable {K} in
+/--
+a non-zero element in an ideal that can be represented as a sum of tensor products of `n`-terms.
+-/
+structure is_obtainable_by_sum_tmul
+    {ιA A B : Type*} [Ring A] [Algebra K A] [Ring B] [Algebra K B]
+    (x : A ⊗[K] B) (𝒜 : Basis ιA K A) (I : TwoSidedIdeal (A ⊗[K] B)) (n : ℕ) : Prop where
+  mem : x ∈ I
+  ne_zero : x ≠ 0
+  rep : ∃ (s : Finset ιA) (_ : s.card = n) (f : ιA → B),
+    x = ∑ i ∈ s, 𝒜 i ⊗ₜ[K] f i
```

Comment:
I'm not convinced that you need this structure at all. The maths proof in [Stacks 074B](https://stacks.math.columbia.edu/tag/074B) should be translated like this:
1. "`W∩(V⊗1)`  is nonzero if `W` is nonzero": convert this to the contrapositive instead, i.e. "`W∩(V⊗1)` = 0 implies `W` = 0".
2. Since you're explicitly using sum of tmuls, translate this to the proposition that given any sum of tmul indexed by `Fin n`, that sum is zero. (This removes the arbitrary indexing type `ιA`.)
3. Prove this new theorem by induction on `n`. The Stacks proof gives you explicitly how to perform the inductive step by reducing the sum.